### ISSUE TITLE
[Feature Request] Add option to supress release notes

### ISSUE THREAD

**Issue by noiob** [telegramdesktop/tdesktop#6641]
Lately a lot of updates have been pushed for this, which is a minor annoyance, but every time I update I have to close the fake "Telegram" chat which only tells me there have been "bug fixes and software optimizations", which is incredibly not useful. Please add an option to supress the release notes or at least not make them show up in the chat list

**Comment by john-preston** [comment]
@noiob Sorry 😬

**Comment by DRSDavidSoft** [comment]
@john-preston Were `1.8.9` through `1.8.13` versions really necessary to release to the public, though?

I had to make sure that the `updater` was not compromised and downloading malware, seeing too many "bug fixes and other minor improvements" updates every other day as Changelog!

https://github.com/telegramdesktop/tdesktop/blob/dcf79df0b2eb0d8bc3757f44c43e62aef9d4fdee/changelog.txt#L1-L20

I wish you could at least make it a bit verbose; seeing that most updates were releated to random app crashes.

**Comment by john-preston** [comment]
@DRSDavidSoft They were very important, but had to be done as soon as possible. I hope it won't be necessary again 🙈

**Comment by DRSDavidSoft** [comment]
@john-preston, In any case, I'd like to thank you for all the work you do!

**Comment by tuxmobil** [comment]
Yes, this is super annyoing. If you have to show a changelog, please don't make it persistent in the chat list. Regardless of how often you do updates. Imagine any other application leaving a photo or document with the changelog behind in your user/home directory. This would be very bad practice.

**Comment by noiob** [comment]
I'm still interested in this

**Comment by DRSDavidSoft** [comment]
No attention from devs = issue closed = doesn't mean the user is not interested = stale bot bad
